### PR TITLE
Re-import css/css-counter-styles/cssom WPT tests

### PR DESCRIPTION
**Pull request by vitorroriz**
#### 28456a7a2c2955df17d584db9da447c92fa0d6ac
<pre>
Re-import css/css-counter-styles/cssom WPT tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=254465">https://bugs.webkit.org/show_bug.cgi?id=254465</a>
rdar://107223884

WPT at a10581823a5de55d896b60ddc60249e7f0e86ff4.
Reviewed by Brent Fulgham.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/cssom/cssom-additive-symbols-setter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/cssom/cssom-fallback-setter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/cssom/cssom-name-setter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/cssom/cssom-negative-setter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/cssom/cssom-pad-setter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/cssom/cssom-prefix-suffix-setter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/cssom/cssom-range-setter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/cssom/cssom-symbols-setter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/cssom/cssom-system-setter-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-counter-styles/cssom/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/262150@main">https://commits.webkit.org/262150@main</a>
</pre>

https://github.com/WebKit/WebKit/commit/28e4c11cf9281c9ab132de9ea6535f78bb6dff95

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/765 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/788 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/816 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/1016 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/682 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/837 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/871 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/907 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/772 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/739 "Passed tests") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/728 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/971 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/795 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/10/builds/721 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/739 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/696 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/743 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/1715 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/728 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/17/builds/707 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/680 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/728 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/172 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/738 "Built successfully") | | | 
| | | | |

### CODE REVIEW

**Review by brentfulgham — APPROVED** [review]
Rs=me

**Comment by brentfulgham** [comment]
It looks like WPE might need some tweaks before landing.

File: LayoutTests/imported/w3c/resources/import-expectations.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "css/css-animations/parsing": "import",
     "css/css-box": "import",
     "css/css-color": "import",
+    "css/css-counter-styles/cssom/": "import",
```
Comment (nt1m) [inline LayoutTests/imported/w3c/resources/import-expectations.json]: Please undo this change, it's already covered below in this file with `"web-platform-tests/css/css-counter-styles": "import",`

File: LayoutTests/TestExpectations [hunk 2]
```diff
@@ -4883,17 +4883,8 @@ imported/w3c/web-platform-tests/css/css-counter-styles/lower-armenian/css3-count
...
-imported/w3c/web-platform-tests/css/css-counter-styles/cssom/cssom-name-setter.html [ ImageOnlyFailure ]
-imported/w3c/web-platform-tests/css/css-counter-styles/cssom/cssom-negative-setter.html [ ImageOnlyFailure ]
-imported/w3c/web-platform-tests/css/css-counter-styles/cssom/cssom-pad-setter.html [ ImageOnlyFailure ]
-imported/w3c/web-platform-tests/css/css-counter-styles/cssom/cssom-prefix-suffix-setter.html [ ImageOnlyFailure ]
-imported/w3c/web-platform-tests/css/css-counter-styles/cssom/cssom-range-setter.html [ ImageOnlyFailure ]
-imported/w3c/web-platform-tests/css/css-counter-styles/cssom/cssom-symbols-setter.html [ ImageOnlyFailure ]
-imported/w3c/web-platform-tests/css/css-counter-styles/cssom/cssom-system-setter-1.html [ ImageOnlyFailure ]
-imported/w3c/web-platform-tests/css/css-counter-styles/cssom/cssom-system-setter-2.html [ ImageOnlyFailure ]
+# Tests that fail because @counter-style is not implemented yet. Skipping directory until rdar://107184142 is implemented.
+imported/w3c/web-platform-tests/css/css-counter-styles/cssom [ Skip ]
```
Comment (nt1m) [inline LayoutTests/TestExpectations:4888]:
```suggestion
imported/w3c/web-platform-tests/css/css-counter-styles/cssom/ [ Skip ]
```

I _think_ you need a trailing `/`, so it's detected as a directory.

Comment (vitorroriz) [reply LayoutTests/TestExpectations:4888]: I had a trailing before and it didn't work so I've removed it as other paths don't have one but still didn't help it. I have to try locally.

Comment (vitorroriz) [reply LayoutTests/TestExpectations:4888]: oh, for some reason the tests are imported to `imported/w3c/css/` instead of `imported/w3c/web-platform-tests/css/`. I'll fix it importing with -d
